cmd/system-shutdown/system-shutdown.c

```diff
@@ -59,6 +59,17 @@ int main(int argc, char *argv[])
 	   oldroot loop device and finally unmount writable itself.
 	 */
 
+        /*
+           We do the sync before anything, as if this exits the kernel should
```

Comment:
I had a bit trouble with the comment, maybe something like: `We do the sync before anything, because if this shutdown helper exits (e.g. in one of the `die()` calls below) before the sync the kernel will panic (...`

Reply:
good idea. I'll do that change before merging.